### DISCUSSION TITLE
IRC Future

### DISCUSSION THREAD

**Discussion by danshick** [any1/wayvnc#103] [General]
I really hate to be that person, but do you intend to keep #wayvnc on Freenode? I wouldn't have felt strongly about any project's decision to stay or leave until this news landed:

https://mastodon.sdf.org/@kline/106299403921451814

I respect whatever decision you make regarding how you manage collaboration on this project, but I figured I'd ask. #wayvnc is the last Freenode channel I am connected to at the moment.

Of the channels I regularly read with a big audience, #pipewire and #wayland moved to OTFC while #sway and #sway-devel have moved to libera.chat.

**Comment by any1** [comment]
No, problem. I've just been focusing on other things recently, so I've been putting off dealing with this. It's probably best to follow the sway crew to libera.

**Reply by danshick** [reply]
Fair enough, no rush.

**Comment by any1** [comment]
The channel has now been registered on libera.chat. See you there!
